File: beecrowd/1074.py
n = int(input())
for c in range(0, n):
    num = int(input())
    if num == 0:
        print('NULL')
    else:
        if num % 2 == 0:
            if num > 0:
                print('EVEN POSITIVE')
            else:
                print('EVEN NEGATIVE')
        else:
            if num % 2 == 1:
                if num > 0:
                    print('ODD POSITIVE')
                else:
                    print('ODD NEGATIVE')

File: beecrowd/1074.c
#include <stdio.h>

int main() {
    
    int testes, num;

    scanf("%d", &testes);

    for (int i = 1; i <= testes; i += 1) {
        scanf("%d", &num);
        if (num == 0)
            printf("NULL\n");
        else {
            if (num % 2 == 0) {
                printf("EVEN ");
            }else {
                printf("ODD ");
            }
            if (num > 0) {
                printf("POSITIVE\n");
            }else {
            printf("NEGATIVE\n");
            }
        }
    }
    return 0;
}